src/Uno.UWP/Graphics/Display/DisplayInformation.Android.cs

```diff
@@ -264,6 +271,20 @@ public DisplayMetricsCache(DisplayMetrics displayMetric)
 				Ydpi = displayMetric.Ydpi;
 			}
 
+			public DisplayMetricsCache(WindowMetrics windowMetric, Android.Content.Res.Configuration? configuration)
+			{
+				HeightPixels = windowMetric.Bounds.Height();
+				WidthPixels = windowMetric.Bounds.Width();
+				if(configuration != null)
+				{
+					Xdpi = configuration.DensityDpi;
+					Ydpi = configuration.DensityDpi;
+					Density = configuration.DensityDpi/160;
+					ScaledDensity = Density;
+				}
+				DensityDpi = DisplayMetricsDensity.Default; // sorry, I don't know how to convert int to DisplayMetricsDensity enum
```

Comment:
This is going to break:
https://github.com/unoplatform/uno/blob/31ba53822d8dae493073913ee11693df64b48ea2/src/Uno.UWP/Graphics/Display/DisplayInformation.Android.cs#L69

We need actual values for Xdpi and Ydpi. Can you [take a look at this](https://github.com/xamarin/xamarin-android/issues/6422#issuecomment-950704742)?